recipes/protobuf/all/conanfile.py

```diff
@@ -109,8 +119,8 @@ def _patch_sources(self):
                 """COMMAND  protobuf::protoc
       ARGS --${protobuf_generate_LANGUAGE}_out ${_dll_export_decl}${protobuf_generate_PROTOC_OUT_DIR} ${_plugin} ${_protobuf_include_path} ${_abs_file}
       DEPENDS ${_abs_file} protobuf::protoc""",
-                """COMMAND "${CMAKE_COMMAND}"  #FIXME: use conan binary component
```

Comment:
Too late now. But you should not have removed this comment. It is a reminder to eventually use an upcoming Conan feature which makes the usage of executables easier.

Reply:
Thanks for the note! Only from my curiosity, can you point towards documentation on that upcoming feature?

Reply:
This might be a start:

https://github.com/conan-io/conan/issues/7240

https://github.com/conan-io/conan/pull/7324